lib/widgets/app.dart

```diff
@@ -149,7 +118,7 @@ class _ZulipAppState extends State<ZulipApp> with WidgetsBindingObserver {
           title: 'Zulip',
           localizationsDelegates: ZulipLocalizations.localizationsDelegates,
           supportedLocales: ZulipLocalizations.supportedLocales,
-          theme: theme,
+          theme: outerThemeData(context),
```

Comment:
The commit message says:

    theme [nfc]: Pull `outerThemeData` out from app.dart to new theme.dart file

but this makes another change too: it computes this ThemeData from within the builder callback for `GlobalStoreWidget.child`. Is that intended?

If the method computing this theme data isn't going to be trying to use the store, then it's probably cleanest to keep the computation outside there, with e.g. `final theme = outerThemeData(context);` at the top of the build method. That way it's clear we compute it once eagerly.

Reply:
Indeed; thanks for noticing this!